Note Datadog Agent version requirement for Vector configuration (#13848)
DataDog/documentation: content/en/agent/vector_aggregation.md

## Configuration

### Agent configuration

> **Note:** below configuration requires Datadog Agent version >= 6/7.35.
> 
> To send logs to Vector, update the Agent configuration file, `datadog.yaml`.
> For logs, update the following values in the `datadog.yaml` file:

Reviewer: ```suggestion
This configuration requires Datadog Agent version >= 6 or 7.35.

To send logs to Vector, update the Agent configuration file, `datadog.yaml`.
```

## Configuration

### Agent configuration

> This configuration requires Datadog Agent version >= 6 or 7.35.

Author: ```suggestion
This configuration requires Datadog Agent version >= 6.35 or 7.35.
```
